inst/examples/safetyGraphics_demo_app.R

```diff
@@ -10,14 +10,109 @@ library(safetyGraphics)
   
-  adam_adlbc <- safetyData::adam_adlbc |>
-    mutate(STRESU = gsub("[\\(\\)]", "", regmatches(PARAM, gregexpr("\\(.*?\\)", PARAM))[[1]]),
-           STRESU = ifelse(PARAM == "Creatinine (umol/L)", "mg/dL", STRESU), # convert creatine to mg/dL
-           AVAL = ifelse(PARAM == "Creatinine (umol/L)", AVAL * .0113, AVAL),
-           PARAM = ifelse(PARAM == "Creatinine (umol/L)", "Creatinine (mg/dL)", PARAM))
+  # Helper functions for unit conversions - shortened names
+  convert_creat_to_mgdl <- function(value) {
+    value / 88.4
+  }
+  
+  convert_bun_to_mgdl <- function(value) {
+    value * 2.86
+  }
+  
+  convert_alb_to_mgdl <- function(value) {
+    value * 100
+  }
+  
+  # Apply unit conversions and data cleaning
+  adam_adlbc_ <- safetyData::adam_adlbc |>
+    mutate(
+      STRESU = gsub("[\\(\\)]", "", regmatches(PARAM, gregexpr("\\(.*?\\)", PARAM))[[1]]),
+      # Use case_when for clearer conditional logic
+      AVAL = case_when(
+        PARAMCD == "CREAT" ~ convert_creat_to_mgdl(AVAL),
+        PARAMCD == "BUN"   ~ convert_bun_to_mgdl(AVAL),
+        PARAMCD == "ALB"   ~ convert_alb_to_mgdl(AVAL),
+        TRUE               ~ AVAL
+      ),
+      BASE = case_when(
+        PARAMCD == "CREAT" ~ convert_creat_to_mgdl(BASE),
+        PARAMCD == "BUN"   ~ convert_bun_to_mgdl(BASE),
+        PARAMCD == "ALB"   ~ convert_alb_to_mgdl(BASE),
+        TRUE               ~ BASE
+      ),
+      CHG = case_when(
+        PARAMCD == "CREAT" ~ convert_creat_to_mgdl(CHG),
+        PARAMCD == "BUN"   ~ convert_bun_to_mgdl(CHG),
+        PARAMCD == "ALB"   ~ convert_alb_to_mgdl(CHG),
+        TRUE               ~ CHG
+      ),
+      A1LO = case_when(
+        PARAMCD == "CREAT" ~ convert_creat_to_mgdl(A1LO),
+        PARAMCD == "BUN"   ~ convert_bun_to_mgdl(A1LO),
+        PARAMCD == "ALB"   ~ convert_alb_to_mgdl(A1LO),
+        TRUE               ~ A1LO
+      ),
+      A1HI = case_when(
+        PARAMCD == "CREAT" ~ convert_creat_to_mgdl(A1HI),
+        PARAMCD == "BUN"   ~ convert_bun_to_mgdl(A1HI),
+        PARAMCD == "ALB"   ~ convert_alb_to_mgdl(A1HI),
+        TRUE               ~ A1HI
+      ),
+      LBSTRESN = case_when(
+        PARAMCD == "CREAT" ~ convert_creat_to_mgdl(LBSTRESN),
+        PARAMCD == "BUN"   ~ convert_bun_to_mgdl(LBSTRESN),
+        PARAMCD == "ALB"   ~ convert_alb_to_mgdl(LBSTRESN),
+        TRUE               ~ LBSTRESN
+      ),
+      STRESU = ifelse(PARAMCD %in% c("BUN", "CREAT", "ALB"), "mg/dL", STRESU),
+      PARAM = case_when(
+        PARAM == "Creatinine (umol/L)"                ~ "Creatinine (mg/dL)",
+        PARAM == "Blood Urea Nitrogen (mmol/L)"       ~ "Blood Urea Nitrogen (mg/dL)",
+        PARAM == "Albumin (g/L)"                      ~ "Albumin (mg/dL)",
+        TRUE                                         ~ PARAM
+      ) # todo: consider removing the units from the PARAM column - if this is implemented also update meta_nepExplorer
```

Comment:
yeah I don't think it affects the app in anyway, but could cause confusion in the documentation and examples if users see that and think "oh I need to have units in the param" when they don't

Reply:
Thanks - lets discuss this further when we look at #155, as we might need to be consistent across the application such as what users see under filters and what is shown on the chart. For example `Creatinine (mg/dL)` vs `Creatinine`